charts/ratify/templates/configmap.yaml

```diff
@@ -61,7 +61,20 @@ data:
                 "verificationCerts": [
                   "/usr/local/ratify-certs/notary/truststore"
                   ],
-                "trustPolicy": "/usr/local/ratify-certs/notary/trustpolicy.json"
+                "trustPolicyDoc": {
+                  "version": "1.0",
+                  "trustPolicies": [
+                    {
```

Comment:
should we update this file too with new trustpolicyDoc section? https://github.com/deislabs/ratify/blob/main/config/config.json

Reply:
Good call! I've updated it as well.